syntax = "proto2";

package evaluation;

message MethodResult {

  required string method_name = 1;
  repeated float num_steps_observed = 2;
  repeated float value_error = 3;
  repeated float num_unvisited_s_a = 4;
  repeated float deterministic_prob = 5;
  repeated float batch_process_num_steps = 6;
  repeated float batch_process_mses = 7;
  // repeated float variances = 4;
  // repeated float mses = 5;
}

message ImprovementResults {

  repeated float dataset_sizes = 1;
  repeated float avg_return = 2;
  repeated float estimated_avg_return = 3;
  repeated float mse = 4;
  optional string label = 5;

}
